### DISCUSSION TITLE
Best folder, on linux, to instal docker version

### DISCUSSION THREAD

**Discussion by GMagician** [blakeblackshear/frigate#14568] [Ask A Question]
Hi all, newby linux user.
I have a fully working container but I'm wondering which is the best location to store frigate docker compose data.
Now I installed on my '/home/user/frigate' but I see that docker works at root level (all folder all built by root and I need to run docker compose with sudo)
N.B. I know I can do something with:
  security_opt:
    - apparmor=unconfined
    - systempaths=unconfined
 but I suspect there is no big difference with root access.

Now I'm wondering may be a good choice to install in in '/opt' folder? or there is a better location to put store it?
Thanks

**Comment by NickM-27** [comment]
I usually just install in /home/user/app data/project_name

**Reply by GMagician** [reply]
thx
